api/openapi/v1/system-agent.yaml

```diff
@@ -8,21 +8,23 @@ paths:
   /v1/config/{services}:
     get:
       description: Fetch the configuration of the specified EdgeX services by their
-        unique names - returning null if none is found for the service. HTTP 500 for
-        unknown or unanticipated issues
+        unique names - returning an error message if a problem is encountered for a
+        given service. And returning HTTP 500 for unknown or unanticipated issues
       parameters:
       - name: services
         in: path
-        description: EdgeX micro service names (the unique name of each service comma
-          delimited)
+        description: EdgeX micro service names (the unique name of each service,
+            comma-delimited)
         required: true
         style: simple
         explode: false
         schema:
           type: string
       responses:
         200:
-          description: an array of config
+          description: an array of elements where each element contains the
+          configuration data themselves, along with the service name each for which
```

Comment:
Got YAML validation failure in the description section. The description contents should display like line 11 if the contents contain multiple lines. Would you please check the full file?